docs/samples/v1beta1/torchserve/v1/config.properties

```diff
@@ -1,10 +1,13 @@
 inference_address=http://0.0.0.0:8085
-management_address=http://0.0.0.0:8081
+management_address=http://0.0.0.0:8085
```

Comment:
is `management_address` supposed to be same as the `inference_address` now?

Reply:
With latest Torchserve release both management and inference APIs can be accessed via same port.